### PR TITLE
WestWind.MarkdownMonster version 2.0.8

### PR DESCRIPTION
**Pull request by RickStrahl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23508)

### CODE REVIEW

**Review by heroku-miraheze — CHANGES_REQUESTED** [review]
What do you mean???

**Comment by RickStrahl** [comment]
So the v2 installer has a second sub-installer for the WebView2 runtime that pops up with UI. I suspect that's what's causing the problem. There's no prompt (ie. it runs without user interaction) but it visually pops up. Could this be the issue?

**Review by ItzLevvie — APPROVED** [review]
image

File: manifests/w/WestWind/MarkdownMonster/2.0.8/WestWind.MarkdownMonster.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: WestWind.MarkdownMonster
+PackageVersion: 2.0.8
```
Comment (ItzLevvie) [inline manifests/w/WestWind/MarkdownMonster/2.0.8/WestWind.MarkdownMonster.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0.8.0
```

File: manifests/w/WestWind/MarkdownMonster/2.0.8/WestWind.MarkdownMonster.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: WestWind.MarkdownMonster
+PackageVersion: 2.0.8
```
Comment (ItzLevvie) [inline manifests/w/WestWind/MarkdownMonster/2.0.8/WestWind.MarkdownMonster.yaml:5]:
```suggestion
PackageVersion: 2.0.8.0
```

File: manifests/w/WestWind/MarkdownMonster/2.0.8/WestWind.MarkdownMonster.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WestWind.MarkdownMonster
+PackageVersion: 2.0.8
```
Comment (ItzLevvie) [inline manifests/w/WestWind/MarkdownMonster/2.0.8/WestWind.MarkdownMonster.installer.yaml:5]:
```suggestion
PackageVersion: 2.0.8.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Oh wait, just for consistency purposes, could you please update the `PackageVersion` key to match what's shown in Control Panel? **Make sure to update the version folder as well, otherwise, it'll fail the validation pipelines.**

If you don't want the `.0` to be there you can update your application to remove it.

**Pull request closed without merging**